src/DotNetLightning.Core/Channel/ChannelTypes.fs

```diff
@@ -297,6 +297,7 @@ type ChannelEvent =
     | Closed
     | Disconnected
     | ChannelStateRequestedSignCommitment
+    | WeReplyToChannelReestablish of msg: ChannelReestablish
```

Comment:
If you update the `FooEvent` in foo domain, you must also update `applyEvent` function in that domain.